onnxruntime/contrib_ops/cpu/quantization/dynamic_quantize_matmul.cc

```diff
@@ -171,6 +280,10 @@ class DynamicQuantizeMatMul final : public MatMulIntegerToFloatBase {
 
  protected:
   int GetBIdx() const override { return IN_B; }
+
+ private:
+  bool CanUseDynamicQuantMLAS{false};
+  bool DynamicQuantMLASPacked{false};
```

Comment:
Can you add some documentation around what each variable mean or the intended purpose behind these vars ?